extensions/control-plane/transfer/transfer-data-plane-signaling/src/test/java/org/eclipse/edc/connector/controlplane/transfer/dataplane/flow/DataPlaneSignalingFlowControllerTest.java

```diff
@@ -66,6 +66,36 @@ public class DataPlaneSignalingFlowControllerTest {
             () -> URI.create("http://localhost"), selectorService, propertiesProvider, dataPlaneClientFactory,
             "random", transferTypeParser);
 
+    @NotNull
+    private DataPlaneInstance.Builder dataPlaneInstanceBuilder() {
```

Comment:
I know that this is likely some automated plugin, but moving private methods above the nested classes that contains the tests causes readability issues.
Automated conventions could be nice, but I would say that in cases like this they could be avoided.

generally speaking, I prefer an approach where "most important stuff" stays at the top, while "least important thing" stays at the bottom.

Reply:
I agree, but the problem with that while it is possible to sort "nested class before private method", there is no code style rule in IntelliJ (to my knowledge) that lets you specify that by class annotation `@Nested`. 

However, it is possible to arrange "nested classes - private/protected method - static nested classes".

Reply:
could it be applied on the other test classes that are involved in the PR?